src/pages/index.js

```diff
@@ -0,0 +1,240 @@
+import plusIcon from "../images/plus.svg";
+import Api from "../utils/Api.js";
+
+document.querySelector(".header__logo").src = logo;
+document.querySelector(".profile__avatar").src = avatar;
+document.querySelector(".profile__edit-btn img").src = pencilIcon;
+document.querySelector(".profile__add-btn img").src = plusIcon;
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "02543b60-f2a8-4764-b91d-a231c86d92a2",
+    "Content-Type": "application/json",
+  },
+});
+
+const cardsListEl = document.querySelector(".cards__list");
+const editProfileBtn = document.querySelector(".profile__edit-btn");
+const editProfileModal = document.querySelector("#edit-profile-modal");
+const editProfileForm = document.forms["edit-profile-form"];
+const editProfileNameInput = editProfileForm.elements["profile-name-input"];
+const editProfileDescriptionInput =
+  editProfileForm.elements["profile-description-input"];
+
+const newPostBtn = document.querySelector(".profile__add-btn");
+const newPostModal = document.querySelector("#new-post-modal");
+const addCardFormElement = document.forms["new-post-form"];
+const nameInput = newPostModal.querySelector("#card-caption-input");
+const linkInput = newPostModal.querySelector("#card-image-input");
+
+const profileNameEl = document.querySelector(".profile__name");
+const profileDescriptionEl = document.querySelector(".profile__description");
+const cardTemplate = document.querySelector("#card-template").content;
+const previewModal = document.querySelector("#preview-modal");
+const previewImage = previewModal.querySelector(".modal__image");
+const previewCaption = previewModal.querySelector(".modal__caption");
+
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarForm = avatarModal.querySelector(".modal__form");
+const avatarFormElement = document.querySelector("#edit-avatar-form");
+const avatarSubmitBtn = avatarModal.querySelector(".modal__submit-btn");
+const avatarModalCloseBtn = avatarModal.querySelector(".modal__close-btn");
+const avatarInput = avatarModal.querySelector("#profile-avatar-input");
+const avatarEditBtn = document.querySelector(".profile__avatar-btn");
+
+function getCardElement(data, currentUserId) {
+  const cardElement = cardTemplate.querySelector(".card").cloneNode(true);
+  const cardImageEl = cardElement.querySelector(".card__image");
+  const cardTitleEl = cardElement.querySelector(".card__title");
+  const cardLikeBtn = cardElement.querySelector(".card__like-btn");
+  const cardDeleteBtn = cardElement.querySelector(".card__delete-btn");
+
+  const cardLikeCount = cardElement.querySelector(".card__like-count");
+
+  cardImageEl.src = data.link;
+  cardImageEl.alt = data.name;
+  cardTitleEl.textContent = data.name;
+  cardLikeCount.textContent = Array.isArray(data.likes) ? data.likes.length : 0;
+
+  if (
+    Array.isArray(data.likes) &&
+    data.likes.some((like) => like._id === currentUserId)
+  ) {
```

Comment:
*  When I reload the page, all **red** hearts become **transparent** again.


You should check `data.isLiked`


```js

  if (data.isLiked) {
    cardLikeBtn.classList.add("card__like-btn_active");
  }

```